laze-project.yml

```diff
@@ -278,8 +278,8 @@ contexts:
         - --cfg context=\"esp\"
         # linkall first
         - -Clink-arg=-Tlinkall.x
-        - -Clink-arg=-Tlinkme-esp-fixup.x
-        - -Clink-arg=-Tlinkme.x
+        # - -Clink-arg=-Tlinkme-esp-fixup.x
+        # - -Clink-arg=-Tlinkme.x
```

Comment:
Commented out because it somehow didn't find the `.rodata` section that is referred to in `linkme.x`. It works right now like this, but I'll try to fix this properly.

Reply:
Out-of band discussion with @kaspar030: we might not need the linkme script for esp32, so it's okay to remove them (correct me if I misunderstood something @kaspar030). Should we then also remove the esp-fixup files?

Reply:
> Should we then also remove the esp-fixup files?

yes, if linkme works without!

Reply:
Tested for esp32c3, esp32c6 and esp32s3 the examples `threading-channel` and `hello-world-async`, works without :+1: